docs/requirements-specification.md

```diff
@@ -57,4 +57,49 @@ _This content item details the use case overview of the system vision in its use
 
 ### UC5: Find expert in a field
 
-### UC6: Organizing capabilities
\ No newline at end of file
+### UC6: Organizing capabilities
+Only Create (C) use cases are explicitly defined here. Update (U) and Delete (D) need still to be defined and shall not contradict/invalidate Create use cases.
+
+| UC6a | Create capability group |
+|---|---|
+| Actors | Admin |If the capability group name exists already, the system prevents the Admin from creating the capability group. |
+| Pre-conditions | The admin is authenticated and has a valid, digital evidence. |
+| End conditions (success) | The capability group is created. |
+| End conditions (failure | The capability group is not created. |
+| Steps | 1. If an Admin presses "Create capability group", the system opens a dialog window with information fields required for a new capability group. |
+| | 2. The Admin enters the information for a new capability group: name, description, and importance (I forgot what we thought about this attribute). |
+| | 3. The system checks whether the capability group name is unique and whether similar capability group names or capability names already exist. |
+| | 4. The Admin presses "Save" and the new capability group is created. |
+| Extensions | 3.a. The system lists similar capability group names and capability names. |
+| | 3.b. If the capability group name exists already, the system prevents the Admin from creating the capability group. |
+| | 4.a. The Admin presses "Cancel" and the new capability group is not created. |
```

Comment:
For completeness, I would add how the system reacts (i.e., does it close the dialog, send the user to a different page, or just show a pop-up?).

Reply:
Hm, I would leave out UI details from the UC. The UC specifies how the system reacts but not how that reaction is implemented concretely in a UI pattern. I would assume other UCs also describe actions, and these should be implemented in a consistent UI pattern.

Reply:
But even from a black-box perspective, the system should be specified to communicate its action to a user. Right now, an implementation that satisfies this UC could just not create a capability group when an admin presses "cancel", but still remain on the same page, leaving the admin to wonder whether the action had success. I would at least make explicit, *that* the system communicates the cancelation, even if we do not say *how* it communicates this.

Reply:
Yes, that makes sense. Will add that to all UCs.